yodapy/datasources/ooi/__init__.py

```diff
@@ -381,11 +381,12 @@ def to_xarray(self, **kwargs):
         # TODO: What to do when it's JSON request, calling on to_xarray.
         # TODO: Standardize the structure of the netCDF to ensure CF compliance.
         # TODO: Add way to specify instruments to convert to xarray
+        ref_degs = self.view_instruments()["reference_designator"].values
```

Comment:
```
def view_instruments(self):
        """
        Shows the current instruments requested.

        Returns:
            DataFrame: Pandas dataframe of the instruments.

        """
        return self._filtered_instruments
```

`view_instruments()` uses `self._filtered_instrument` so please change to use that instead.